state_machine/applications/system-check/tests/coil_test.py

```diff
@@ -0,0 +1,146 @@
+"""
+Python system check script for PyCubed satellite board
+PyCubed Mini mainboard-v02 for Pocketqube Mission
+Torque Driver Test
+* Author(s): Yashika Batra
+"""
+
+import time
+from lib import pycubed as cubesat
+from ulab import numpy
+
+# voltage level constants; set between -1 and 1
+v1 = 0.25
+v2 = 0.5
+v3 = 0.75
+
+# find projected voltage: vlevel * max voltage
+projected_voltage1 = v1 * 5.06
+projected_voltage2 = v2 * 5.06
+projected_voltage3 = v3 * 5.06
+
+# alias norm
+norm = numpy.linalg.norm
+
+
+def test_voltage_levels(coil_index):
+    """
+    Ask the user to type y/any key to start/cancel the test
+    If y, collect IMU magnetometer data for different voltage levels,
+    and return if the magnetometer readings are increasing as voltage
```

Comment:
Assuming the generated magnetic field is actually near the 1000 µT threshold, this shouldn't be an issue, as the Earth's magnetic field is small (~30-60 µT depending on location) in comparison.

If the threshold is smaller (as we discussed in another comment), this would probably pose an issue.

Reply:
Yeah given your comment, I think we're pretty safe.